crates/hdk/src/hash_path/anchor.rs

```diff
@@ -85,20 +82,28 @@ impl TryFrom<&Path> for Anchor {
 
 /// Simple string interface to simple string based paths.
 /// a.k.a "the anchor pattern" that predates paths by a few years.
-pub fn anchor(anchor_type: String, anchor_text: String) -> ExternResult<holo_hash::EntryHash> {
+pub fn anchor(
+    link_type: impl Into<LinkType>,
+    anchor_type: String,
+    anchor_text: String,
+) -> ExternResult<holo_hash::EntryHash> {
```

Comment:
Kind of a side concern, but I wonder if anchors still need an anchor type if they already have a link type? @artbrock

Reply:
Hmm... well... anchor type is part of the hashed content (or first path element)...  
hash(AnchorType + AnchorText). Seems like we lose part of the hashed data, unless you are suggesting that we have to make a different LinkType for every AnchorType... and then we could just put the LinkType into the hashing algo.